lib/fictium/exporters/api_blueprint_exporter/action_formatter.rb

```diff
@@ -0,0 +1,31 @@
+module Fictium
+  class ApiBlueprintExporter
+    class ActionFormatter < Fictium::ApiBlueprintExporter::BaseFormatter
+      protected
+
+      def format_sections(action)
+        [build_header(action), build_examples(action)]
+      end
+
+      private
+
+      def build_header(action)
+        result = "## #{action.summary} [#{action.method.to_s.upcase} #{action.path}]"
+        "#{result}#{action.description.present? ? "\n\n#{action.description}\n" : ''}"
```

Comment:
This line is a little harder to read... I mean, in a simple view you saw this
```
"#{result}#{action.description.present? ? "
```
And this is not the line to process... The interpreter can do a well job for process this, but maybe you can do it more readable like this:
```
        result = "## #{action.summary} [#{action.method.to_s.upcase} #{action.path}]"
        description = action.description.present? ? "\n\n#{action.description}\n" : ''
        "#{result}#{description}"
```
The same apply to the method build_header in lib/fictium/exporters/api_blueprint_exporter/resource_formatter.rb

What do you think @holywyvern ?

Reply:
understandable